go/def.bzl

```diff
@@ -558,6 +578,15 @@ go_library_attrs = go_env_attrs + {
     ),
 }
 
+go_tool_attrs = {
+    "filter_tags": attr.label(
```

Comment:
I think this should be added to go_env_attrs instead. It should be a private attribute ("_filter_tags"). I don't think anyone would want to override this.